frontend/src/features/permits/helpers/mappers.ts

```diff
@@ -289,3 +289,19 @@ export const transformApplicationToPermit = (
     },
   };
 };
+
+/**
+ * Remove empty values from permits action response
+ * @param res Permits action response received from backend
+ * @returns Permits action response having empty values removed
+ */
+export const removeEmptyIdsFromPermitsActionResponse = (
+  res: PermitsActionResponse,
+): PermitsActionResponse => {
+  const successIds = getDefaultRequiredVal([], res.success).filter(id => !!id);
+  const failedIds = getDefaultRequiredVal([], res.failure).filter(id => !!id);
```

Comment:
```suggestion
  const successIds = getDefaultRequiredVal([], res.success).filter(id => Boolean(id));
  const failedIds = getDefaultRequiredVal([], res.failure).filter(id => Boolean(id));
```
Makes it a bit more readable. Ignore if you prefer current way.

Reply:
Ok I'll change it